source/reference/api/progress.txt

```diff
@@ -28,11 +28,24 @@ Request
 Response
 --------
 
-All response fields are wrapped in a top-level ``progress`` object. The
-endpoint returns either an updated status or an error.
+The ``progress`` endpoint returns etiher an updated status or an error.
+
+Successful Response
+~~~~~~~~~~~~~~~~~~~
+
+On a successful return, all response fields are wrapped in a top-level
+``progress`` object with the following fields:
 
 .. include:: /includes/api/tables/progress-response.rst
 
+Error Response
+~~~~~~~~~~~~~~
+
+If ``mongosync`` encounters an error, the ``progress`` endpoint instead
```

Comment:
[nit, minimalism]

I think we can remove "instead".

Reply:
Fixed